gdal/swig/python/osgeo/utils/auxiliary/color_palette.py

```diff
@@ -352,13 +352,10 @@ def get_color_palette(color_palette_or_path_or_strings: ColorPaletteOrPathOrStri
     return pal
 
 
-def test_xml():
-    dir_path = 'sample/color_files'
+def test_xml(dir_path):
```

Comment:
Is this actually used? AFAICS the only caller was below

Reply:
Thanks for your comments.
I was planning to add a test to use this function with the actual test files but I didn't get to it yet.
Meanwhile I wanted to replace the reference to this hardcoded dir which only existed in my local setup.

Reply:
Suggest to drop it out and re-add on the appropriate future PR.

Reply:
> Suggest to drop it out and re-add on the appropriate future PR.

done. thanks!

Reply:
[flake8 is failing](https://github.com/OSGeo/gdal/pull/3404/checks?check_run_id=1862571265) on `import glob` being unused

Reply:
Thanks